recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,3 @@
+pytorch:
+  - '1.12'
```

Comment:
Was it too hard to build 1.11? Didn't you need that version?

Reply:
ahah I am confused now. I understood it was too much burden for you to build for many torch versions.

As for my specific need I think >=1.12 should be fine.

Also I spotted some numerical errors due to 1.11 making the tests to fail and I am a bit lazy to look into that right now.

Given the above, I propose to skip 1.11 for now and I will rebuild for it in the future if I really need it.

Reply:
Any reason you need to build for two pytorch versions? Can't we just let the migrators take care of this?

Reply:
It's not urgent for me. I was actually wondering that at https://github.com/conda-forge/pyg-lib-feedstock/issues/2#issuecomment-1371267833

Reply:
It is too much for me to builld old torch versions with new dependencies. Eventually, you won't be in a position to build new packages, with new pinnings, with the old pytorch packages.

But if it works, and you need refreshed packages for whatever reason, then this approach is fine so long as it continues to work.

Reply:
Ok i will merge here then.

I removed the 1.13 builds and will wait for the bot to bump it.